src/database.sql

```diff
@@ -0,0 +1,16 @@
+--psql efosdan
+--\c teamworkdb
```

Comment:
i want this to be a js file so you can have a script and you can run npm run migrateDb and it will run the file